cloudflare_exporter/collector.py

```diff
@@ -5,43 +5,46 @@
 import CloudFlare
 from prometheus_client.core import GaugeMetricFamily
 
-from cloudflare_exporter.config import (LOGS_EXTENDED_METRIC,
-                                        LOGS_EXTENDED_METRIC_COUNT,
-                                        LOGS_EXTENDED_METRIC_SAMPLE,
-                                        LOGS_EXTENDED_METRIC_RANGE)
 from cloudflare_exporter.lib.statistics import quantiles
 
 
 class CloudflareCollector:
-    def __init__(self, cloudflare_token):
+    # pylint: disable-msg=too-many-arguments
+    def __init__(self, cloudflare_token, logs_extended_metric, logs_count, logs_sample, logs_range):
         self.cloudflare_token = cloudflare_token
+        self.logs_extended_metric = logs_extended_metric
+        self.logs_count = logs_count
+        self.logs_sample = logs_sample
+        self.logs_range = logs_range
 
     def logs_collect(self, families):
```

Comment:
Some docstring explaining what this function does (extract 50th and 90th response time percentile) could be helpful.

Reply:
done